cms/templates/studio_xblock_wrapper.html

```diff
@@ -64,6 +64,8 @@
     use-new-editor-video = ${use_new_editor_video}
     use-new-editor-problem = ${use_new_editor_problem}
     authoring_MFE_base_url = ${get_editor_page_base_url(xblock.location.course_key)}
+    data-block-type = ${xblock.scope_ids.block_type}
```

Comment:
I'm curious: `data-block-type` is never referenced in this file again, why are we adding it here?

Reply:
So I am adding it here to the "xblock-header-primary" HTML Tag. Before, this information was extracted by the JS Frontend from the xblock itself, but the xblock didn't exist. I am adding this information to the header so that the JS Frontend has access to it, can make the necessary check to redirect to the new editor, and then use this info also to construct the URL.

Reply:
Ah now I see the difference in the code. Thanks for clearing that up.